core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.component.ts

```diff
@@ -101,6 +102,7 @@ angular.module('oppia').component('feedbackTab', {
             result.newThreadSubject, result.newThreadText)
         ).then(() => {
           ctrl.clearActiveThread();
+          ctrl.$onInit();
```

Comment:
Hi @nithusha21! So sorry for such a late reply. Actually, I tried working on this issue but nothing came up to my mind and later on, I started working on some other issue. I am thinking of dropping this PR for the time being because I am working on a GSOC idealist project for adding a stats page for contributors dashboard and on some other automated QA team issue too. Later on, we can discuss this. If it is some easy fix then you can please share the approach, I will try to work on this PR simultaneously and raise another PR whenever I get free from the automated QA team issue.

Reply:
I do want to learn how we can create a frontend cache. Rijuta told me that we are doing the same thing for explorations too. So if you can share some reference code or something similar in our codebase then it would be great!

Reply:
@Rijuta-s can you please link the commit or PR for the equivalent of this issue, but for explorations? (as described in the last [ayushjaink8](https://github.com/ayushjaink8) comment) thanks!